server/server.go

```diff
@@ -102,6 +101,17 @@ func (s *Server) Stop() {
 	manners.Close()
 }
 
+func (s *Server) RefreshMembers() {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+	members, err := s.getMembers(ctx)
+	if err != nil {
+		s.logErrorToMattermost(err.Error())
+		mlog.Error("failed to refresh org members", mlog.Err(err))
+	}
```

Comment:
The code should return here to ensure that `s.OrgMembers` doesn't get overwritten.
```suggestion
		mlog.Error("failed to refresh org members", mlog.Err(err))
		return
	}
```